mattermost-plugin/webapp/src/index.tsx

```diff
@@ -32,6 +36,9 @@ import {PluginRegistry} from './types/mattermost-webapp'
 
 import './plugin.scss'
 
+const TELEMETRY_RUDDER_KEY = 'placeholder_rudder_key'
```

Comment:
Right now, `placeholder_rudder_key` is replaced in the [production build workflow](https://github.com/mattermost/focalboard/blob/main/.github/workflows/prod-release.yml#L14) only in `telemetry.go`. Looks like we either need to update all the build workflows to replace it in this file as well, or expose and call [getRudderConfig](https://github.com/mattermost/focalboard/blob/main/server/services/telemetry/telemetry.go#L53) in telemetry?